service/controller/key/provider_appspecs.go

```diff
@@ -33,7 +33,7 @@ func AWSAppSpecs() []AppSpec {
 			ClusterAPIOnly:  true,
 			Namespace:       metav1.NamespaceSystem,
 			UseUpgradeForce: true,
-			Version:         "1.0.0",
+			Version:         "1.0.1",
```

Comment:
@corest I think this is the only version bump. 

Deploys https://github.com/giantswarm/cert-manager-app/pull/3 by @piontec